Gemfile.lock

```diff
@@ -780,7 +780,7 @@ DEPENDENCIES
   web-console (~> 3.5)
 
 RUBY VERSION
-   ruby 2.6.3p62
+   ruby 2.6.5p114
```

Comment:
This change is already in master no? why are you doing it here?

Reply:
mhhh
In master, the ruby version is `2.6.5` in .ruby-version...
I have only installed the Ruby version in localhost and I run `bundle install`

Reply:
Then I would undo the change, merge master and `bundle install` again. This should not be updated by bundler unless you are running a different ruby version. Maybe you changed it at a shell level?

Reply:
Sorry, the ruby version in master is `2.6.3` Check https://github.com/decidim/decidim/blob/master/Gemfile.lock#L782

Reply:
Done!